packages/sw-background-sync-queue/test/unit/response-manager.js

```diff
@@ -0,0 +1,50 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/* eslint-env mocha, browser */
+/* global chai, goog */
+
+'use strict';
+
+describe('response-manager test', () => {
+  const response = 'VALUE';
+  const idbHelper = new goog.backgroundSyncQueue.test.IDBHelper(
```

Comment:
`goog.backgroundSyncQueue.test` is the namespace where all the libs are compiled and put for testing separately